addon/templates/components/polaris-avatar.hbs

```diff
@@ -15,12 +15,15 @@
   </span>
 {{/if}}
 
-{{#if finalSource}}
+{{#if shouldShowImage}}
   {{!-- TODO: replace this with polaris-image when implemented. --}}
   <img
+    {{! template-lint-disable no-invalid-interactive }}
     class="Polaris-Avatar__Image"
     src={{finalSource}}
     alt=""
     role="presentation"
+    onload={{action "handleLoad"}}
```

Comment:
just double-checking these are the right events here in Ember world

Reply:
Well, I initially went with `{{action "handleLoad" on="load"}}` element modifiers and tried tinkering with that, but apparently that approach doesn't work here so this is what I landed on. The updated tests cover this functionality and with the exception of a timing issue in the tests for Ember 2.16/2.18, they pass without problem.